qa/common/gen_qa_custom_ops

```diff
@@ -73,19 +73,19 @@ TF_LFLAGS=\$(python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get
 # No CUDA
 cd /tmp
 cp /opt/tensorflow/tensorflow-source/tensorflow/examples/adding_an_op/zero_out_op_kernel_1.cc .
-g++ -std=c++11 -O2 -shared -fPIC zero_out_op_kernel_1.cc -o $DESTDIR/libzeroout.so \${TF_CFLAGS[@]} \${TF_LFLAGS[@]}
+g++ -std=c++14 -O2 -shared -fPIC zero_out_op_kernel_1.cc -o $DESTDIR/libzeroout.so \${TF_CFLAGS[@]} \${TF_LFLAGS[@]}
```

Comment:
were these intended to be updated to c++14? is it required for something? same as below.

Reply:
There will be compiled issues on tf2 images if using C++11. Using C++14 seems to solve those syntax problems.

Reply:
@tanmayv25 @GuanLuo do we have any constraints to use C++11? Or this is OK?

Reply:
I don't have strong opinion on the C++ version to be used. And I don't think it matters here because it is only for custom operator which the user can compile it with any flavor independent from Triton.

Reply:
C++14 is fine.